conduit/test/main.hs

```diff
@@ -235,6 +236,20 @@ main = hspec $ do
                     C.=$ CL.fold (+) 0
             x `shouldBe` 2 * sum [1..10 :: Int]
 
+        it "chunksOf" $ do
+            let input = [1..16] :: [Int]
```

Comment:
How about turning this into a `prop` with a variable size? Or add that as an additional case?